nemo_curator/scripts/find_pii_and_deidentify.py

```diff
@@ -92,89 +90,60 @@ def attach_args(
         formatter_class=argparse.ArgumentDefaultsHelpFormatter,
     )
 ):
+    argumentHelper = ArgumentHelper(parser)
 
-    parser.add_argument(
-        "--language",
-        type=str,
-        default="en",
-        required=False,
-        help="Language of input documents",
+    argumentHelper.add_arg_batch_size(
+        default=2000, help="The batch size for processing multiple texts together."
     )
-
-    parser.add_argument(
-        "--supported-entities",
-        type=str,
-        default=None,
-        required=False,
-        help="Comma separated list of PII entity types. None implies all supported types",
+    argumentHelper.add_arg_input_data_dir(help="Directory containing the input files.")
+    argumentHelper.add_arg_input_file_type(
+        choices=["jsonl", "csv", "text"],
```

Comment:
Hi Ryan,

I am not sure to understand it. arguentHelper.add_arg_input_file_type() method default value is "jsonl", which I think is the one that was originally in the code. Am I missing something? I also removed the `required=True` parameter because we had a default value, so no need to specify both.

Am I missing something? Thanks!

Reply:
Sorry, let me clarify. You did correctly copy over the original arguments from the file. I am pointing out that the original arguments were actually incorrect haha. Both `choices` and `help` should remain with the default values you have given them. Therefore, this function call should look like `argumentHelper.add_arg_input_file_type()` and the output file type should also be `argumentHelper.add_arg_output_file_type()`.

This is because in the script where `args.input_file_type` is used, the `read_data` function is called which actually supports the usual file types (`jsonl`, `parquet`, etc.) The `help` text is also incorrect for this function call since other file types are supported.

This mistake was easy to spot for me because of your changes, so it looks like this is proof your changes are going to be a big help. :)